src/plugins/dashboard/public/application/top_nav/editor_menu.tsx

```diff
@@ -239,10 +246,10 @@ export const EditorMenu = ({ dashboardContainer, createNewVisType }: Props) => {
       panelPaddingSize="none"
       data-test-subj="dashboardEditorMenuButton"
     >
-      {() => (
+      {({ closePopover }: { closePopover: () => void }) => (
```

Comment:
Note that, before this, the `SolutionToolbarPopover` was **only** used for listing all of the visualization types - however, since the `onClick` of these options often leads to a different app, the popover remaining open didn't really matter. The **only** instance where this bug was present before this change was when adding `Log stream` visualizations - since this doesn't navigate to another app, you'll notice that the popover stayed open before this fix,